src/operations/support/editorUtils.ts

```diff
@@ -92,7 +92,7 @@ function doWithEditResult(r: EditResult<GitProject>, gitop: () => Promise<EditRe
                 }) : gitop();
             });
     }
-    logger.info("NOT committing %j as it's not dirty, edited=%s", r.target.id, r.edited);
+    logger.info("NOT committing %j in %s as it's not dirty, edited=%s", r.target.id, r.target.baseDir, r.edited);
```

Comment:
I think we should add logging on all the branches in this function. It's important information and will be invaluable for debugging

Reply:
ok. yeah this came up for us as I was working on this so I threw it in